src/main/java/com/corndel/cashregister/exercises/Exercise2b.java

```diff
@@ -9,7 +9,6 @@ public class Exercise2b {
    * name of each recipe
    */
   public static List<String> listNames(List<Recipe> recipes) {
-    // TODO
-    return null;
+    return recipes.stream().map(recipe -> recipe.getName()).toList();
```

Comment:
Great use of `.map()`!

You could use a Method reference instead of passing a Lambda function. 

```java
return recipes.stream().map(Recipe::getName).toList();
```